packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -398,7 +444,7 @@ class AnimationController extends Animation<double>
     }
     assert(simulationDuration > Duration.zero);
     assert(!isAnimating);
-    return _startSimulation(new _InterpolationSimulation(_value, target, simulationDuration, curve));
+    return _startSimulation(new _InterpolationSimulation(_value, target, simulationDuration, curve, scale));
```

Comment:
why not just multiply the duration here rather than pass the value and then have the constructor multiply?